components/VM_Arm/src/main.c

```diff
@@ -826,6 +856,19 @@ static int load_linux(vm_t *vm, const char *kernel_name, const char *dtb_name, c
             paths = camkes_dtb_get_node_paths(&num_paths);
         }
 
+        /* If dtb_base_name is in the file server, grab it and use it as a base */
+        if (open(linux_image_config.dtb_base_name, 0)) {
+            size_t dtb_len = get_file(linux_image_config.dtb_base_name, linux_gen_dtb_base_buf,
```

Comment:
Can get_file just be replaced with: `size_t dtb_len = read(dtb_fd, linux_gen_dtb_base_buf, PLAT_LINUX_DTB_SIZE);`?
Using the fd returned by the open call above, and then followed with a close() after.

Reply:
Sure. That cleans up the commit nicely.